modules/gapi/include/opencv2/gapi/render.hpp

```diff
@@ -50,12 +50,41 @@ struct Rect
 {
     cv::Rect   rect;  /** Coordinates of the rectangle < */
     cv::Scalar color; /** The rectangle color or brightness (grayscale image) < */
```

Comment:
I believe this `< ` thing is not required actually? Please take a look how other public structure fields are documented in OpenCV. When I checked the generated documentation for OSD the last time, I saw that `< ` there in the generated comments.

Reply:
Fixed

Reply:
![image](https://user-images.githubusercontent.com/144187/60025512-772fcb00-96a2-11e9-9fdb-e400e86cf78d.png)

Much nicer now, thanks!